vsdeinterlace/qtgmc.py

```diff
@@ -1146,60 +1131,69 @@ def _apply_sharpen(self, clip: vs.VideoNode) -> vs.VideoNode:
                 )
                 resharp = unsharpen(
                     clip,
-                    self.sharp_strength[1],
-                    BlurMatrix.BINOMIAL(self.sharp_strength[0])(clamp, func=self._apply_sharpen),
+                    self.sharpen_strength,
+                    BlurMatrix.BINOMIAL()(clamp, func=self._apply_sharpen),
                     func=self._apply_sharpen,
                 )
 
-        if self.sharp_thin:
+        if self.sharpen_thin:
             median_diff = norm_expr(
                 [clip, median_blur(clip, mode=ConvMode.VERTICAL)],
                 "y x - {thin} * neutral +",
-                thin=self.sharp_thin,
+                thin=self.sharpen_thin,
                 func=self._apply_sharpen,
             )
             blurred_diff = BlurMatrix.BINOMIAL(mode=ConvMode.HORIZONTAL)(median_diff, func=self._apply_sharpen)
 
             resharp = norm_expr(
-                [resharp, blurred_diff, BlurMatrix.BINOMIAL()(blurred_diff)],
+                [resharp, BlurMatrix.BINOMIAL()(blurred_diff), blurred_diff],
                 "y neutral - dup abs z neutral - abs < swap x + x ?",
                 func=self._apply_sharpen,
             )
 
         return resharp
```

Comment:
This variable is unbound when `sharpen_strength` is falsy and `sharpen_thin` is truthy